### ISSUE TITLE
IAP Homepage: (temporary fix) H1 missing and image of text doesn't include all required alt text

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#4718]
## Description

Note that the following is a temporary fix (ideally images of text should not be used).  See issue #4719  for a more permanent solution.

There are two issues with the image of text for the logo on this page.

1. There is no `<h1>` on the page.
Add hidden `<span>` instead of alt text for "IT Apprenticeship Program for Indigenous Peoples" with the logo image.  Wrap the hidden text in an `<h1>` for screen reader users.

2. Subtitle text was not included in the alt text.
Add "Apply today to get started on your IT career journey" as part of alternative text for screen reader users.

## Screenshot: Prototype
<img width="738" alt="Screen Shot 2022-11-10 at 5 14 46 PM" src="https://user-images.githubusercontent.com/1794660/201217799-6d03af3e-6ee7-4809-9663-60739f4b7770.png">

## Acceptance Criteria
- [ ] ...

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap
